lint.go

```diff
@@ -1159,11 +1159,17 @@ func (f *file) lintErrors() {
 	}
 }
 
-func lintCapAndPunct(s string) (isCap, isPunct bool) {
+func lintCapAndPunct(s string) (isCap, isPunct, isNewline bool) {
```

Comment:
My suggestion below about having a single message for all violations makes it so that we don't need to differentiate between isCap, isPunct, isNewline.

Furthermore, we can probably rename lintCapAndPunct to lintErrorString or something.